static/templates/admin_user_group_list.handlebars

```diff
@@ -4,5 +4,8 @@
     <div class="pill-container" data-group-pills="{{name}}">
         <div class="input" contenteditable="true"></div>
     </div>
+    <button class="button rounded small delete btn-danger">
+        <i class="icon-vector-trash" aria-hidden="true"></i>
```

Comment:
Can you please change the `class="icon-vector-trash"` to `class="fa fa-trash"`. `icon-vector` was part of an obsolete standard of the font awesome library which they deprecated around v3.0 of FA. We have been meaning to remove all the `icon-vector` occurrences and use latest FA only. Currently everything from FA v4.0 can be used in Zulip.